weave/trace_server/model_providers/model_providers.json

```diff
@@ -1 +1 @@
y_name": "BEDROCK_API_KEY"}, "bedrock/eu-west-1/meta.llama3-70b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "bedrock/eu-west-2/meta.llama3-70b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "bedrock/sa-east-1/meta.llama3-70b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "meta.llama3-1-8b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "us.meta.llama3-1-8b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "meta.llama3-1-70b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "us.meta.llama3-1-70b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "meta.llama3-1-405b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "us.meta.llama3-1-405b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "meta.llama3-2-1b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "us.meta.llama3-2-1b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "eu.meta.llama3-2-1b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "meta.llama3-2-3b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "us.meta.llama3-2-3b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "eu.meta.llama3-2-3b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "meta.llama3-2-11b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "us.meta.llama3-2-11b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "meta.llama3-2-90b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "us.meta.llama3-2-90b-instruct-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "512-x-512/50-steps/stability.stable-diffusion-xl-v0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "512-x-512/max-steps/stability.stable-diffusion-xl-v0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "max-x-max/50-steps/stability.stable-diffusion-xl-v0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "max-x-max/max-steps/stability.stable-diffusion-xl-v0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "1024-x-1024/50-steps/stability.stable-diffusion-xl-v1": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "1024-x-1024/max-steps/stability.stable-diffusion-xl-v1": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "stability.sd3-large-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}, "stability.stable-image-ultra-v1:0": {"litellm_provider": "bedrock", "api_key_name": "BEDROCK_API_KEY"}}
```

Comment:
I wonder if we could format this file in a way that we could see what changes between versions?